uber/models/__init__.py

```diff
@@ -2250,24 +2249,14 @@ def initialize_db(modify_tables=False):
     drop=True or modify_tables=True or initialize=True to
     Session.initialize_db()
     """
-    num_tries_remaining = 10
```

Comment:
Why was retrying DB connection removed?

Reply:
Good point, I ripped it out as part of removing the stopped and on_startup events but I could put it in a for loop with a dozen retries or something. In prod the whole process will just restart if it dies, but this could be a much cheaper loop than that.